mycore-ocfl/src/main/java/org/mycore/ocfl/commands/MCROCFLCommands.java

```diff
@@ -121,6 +126,50 @@ public static List<String> syncClassificationRepository() {
         return commands;
     }
 
+    @MCRCommand(syntax = "update ocfl users",
+        help = "Update all users in the OCFL store from database")
+    public static List<String> updateOCFLUsers() {
+        List<MCRUser> list = MCRUserManager.listUsers("*", null, null, null);
+
+        return list.stream()
+            .map(usr -> "update ocfl user " + usr.getUserID())
+            .collect(Collectors.toList());
+    }
+
+    @MCRCommand(syntax = "update ocfl user {0}",
+        help = "Update user {0} in the OCFL Store from database")
+    public static void updateOCFLUser(String userId) {
+        if (MCRUserManager.getUser(userId) == null) {
+            throw new MCRUsageException("The User '" + userId + "' does not exist!");
+        }
+        new MCROCFLXMLUserManager().updateUser(MCRUserManager.getUser(userId));
+    }
+
+    @MCRCommand(syntax = "delete ocfl user {0}",
+        help = "Delete user {0} in the OCFL Store")
+    public static void deleteOCFLUser(String userId) {
+        new MCROCFLXMLUserManager().deleteUser(userId);
+    }
+
+    @MCRCommand(syntax = "sync ocfl users",
+        help = "Update all users and remove deleted users to resync OCFL Store to the Database")
+    public static List<String> syncUserRepository() {
+        List<String> commands = new ArrayList<>();
+        commands.add("update ocfl users");
+        List<String> outOfSync = getStaleOCFLUserIDs();
+        commands.addAll(
+            outOfSync.stream()
+                .map(id -> "delete ocfl user " + id).collect(Collectors.toList()));
+        return commands;
+    }
+
+    @MCRCommand(syntax = "restore database user {0} with version {1} from ocfl",
```

Comment:
I think `restore user {0} with version {1} from ocfl` suits this best, since the "ocfl" could be seen as parameter.

> Or even better to match the other new commands: "restore ocfl user {0} with version {1}:

This, at least for me, implies that the user within ocfl gets restored, so it could be misleading.

Reply:
O.K. If we see "ocfl" as parameter that's fine with me. 
I would still see the version parameter at the end of the command, because the version is specific to the OCFL repository.
But finally it's your decision.